lucene/core/src/java/org/apache/lucene/index/EagerCheapMergePolicy.java

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
Can we move this to sandbox instead?

Reply:
Why do you think?  It'd be nice for users to get an implementation of findFullFlushMerges without requiring sandbox.

Reply:
I think this a rather dangerous merge policy, at least as I currently understand it, so I think baking for a while in `sandbox` so we can confirm for more general users this is indeed safe.